.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,13 +19,12 @@ Optional. Screenshots, `curl` examples, etc.
 
 Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.
 
-
 ## Testing Instructions
 
- * How to test this PR
- * Prefer bulleted description
- * Start after checking out this branch
- * Include any setup required, such as bundling scripts, restarting services, etc.
- * Include test case, and expected output
+- How to test this PR
```

Comment:
I believe more Prettier changes to make bullet syntax consistent within the document.